src/blockchain/database/Headers.cpp

```diff
@@ -41,6 +41,7 @@
 #include "serialization/protobuf/BlockchainBlockLocalData.pb.h"
 #include "util/LMDB.hpp"
 #include "util/Work.hpp"
+#include "util/threadutil.hpp"
```

Comment:
Mostly no real changes here

Reply:
Here it is needed for MessageMarker, lines 227. 235.